server/usingDB/controllers/user-controller.js

```diff
@@ -0,0 +1,49 @@
+import helper from '../auth/auth';
+import db from '../../db';
+import { STATUS_CODES } from 'http';
+
+const User = {
+    async signupUser(req, res) {
+        //const newUser = {...req.body};
+
+        const hashPassword = helper.hashPassword(req.body.password);
+
+        const createUserQuery = `INSERT INTO users(firstName, lastName, email, hashpassword)
```

Comment:
you can create a query helper file where all your query will be located, and you call them where you want to use them